packages/cypress/src/executors/cypress/schema.json

```diff
@@ -39,7 +39,7 @@
     },
     "devServerTarget": {
       "type": "string",
-      "description": "Dev server target to run tests against."
+      "description": "Dev server target(s) to run tests against. Multiple dev server targets can be specified with comma separated values, e.g: 'app1:target1:config1, app2:target2:config2', of which last dev server target yielded baseUrl will be passed over to Cypress."
```

Comment:
issue: this should be a string array instead of a comma separated list. when making into a string array. all of component testing will need to be refactored to make sure they are reading the correct target. and a migration should be added to update this property. 

optionally a string or string array could be used. so then a migration wouldn't need to be written, but an extra check for component testing will still needed to be taken.

Reply:
I was trying to make things flexible and backwards compatible by keeping it as a string, so no migration is needed. For most use cases, single dev server is all they need.